tests/source/configs/comment_width/5801.rs

```diff
@@ -0,0 +1,16 @@
+// rustfmt-comment_width: 120
```

Comment:
If there's not change between the input and the output then you can simple remove the `source` file.

Reply:
There is no change, I will drop the `/source` file